app/logic/events.py

```diff
@@ -140,10 +141,44 @@ def groupEventsByCategory(term):
                          "One Time Events" : groupEventsByProgram(oneTimeEvents)}
     return categorizedEvents
 
+def eventEdit(newEventData):
```

Comment:
It concerns me that this is in here. Is this from a merge conflict? It is really old, and wrong.